### DISCUSSION TITLE
Carillon for GrandOrgue?

### DISCUSSION THREAD

**Discussion by rudike6** [GrandOrgue/grandorgue#1477] [Help]
Hello, is there ANY carillon sampleset available for grandorgue?

a year or what ago there was Carillon de Gandt. Ik can't find it anymore? Is it still available for download?
Or, can anybody share the sampleset for download here?

**Comment by eturpault** [comment]
There are the audio samples of the Lurie carillon at university of Michigan which can be downloaded here : https://gobluebells.wordpress.com/2016/09/21/lurie-carillon-samples/
It has 60 bells, the first one sings at MIDI note 43.
These samples have better quality and duration than the one of the Carillon of Gandt.
I built a sample set with .organ file to play these samples in GrandOrgue, I share it at this link : https://drive.google.com/file/d/1LCkJrOIq9ffRL36pKuOYToyDOrj-P9Nn/view?usp=sharing

**Reply by rudike6** [reply]
Thank you! I'll try it.

**Reply by eturpault** [reply]
@larspalo : if you agree, the sample set that I made with the samples of the Lurie carillon of the Michigan university could be proposed in this web page : https://github.com/GrandOrgue/grandorgue/wiki/Sample-Sets.
I made today some improvements in the ODF, so here is the link to download the new archive file containing both the ODF and the samples : https://drive.google.com/file/d/10Bg-xufE7jJKFIb0JqfJdPNjByjd5JrH/view?usp=sharing

**Reply by larspalo** [reply]
Done!

**Comment by eturpault** [comment]
Another carillon : https://sourceforge.net/p/ourorgan/samplesets/451-Carillon%2049%20notes%20/

**Reply by rudike6** [reply]
The last carillon has no downloadlink and the 'listen' function doesn't work.
